cnf-certification-test/accesscontrol/suite.go

```diff
@@ -185,16 +205,55 @@ var _ = ginkgo.Describe(common.AccessControlTestKey, func() {
 	})
 })
 
-// TestSecConCapabilities verifies that non compliant capabilities are not present
-func TestSecConCapabilities(env *provider.TestEnvironment) {
+func TestSysAdminCapability(env *provider.TestEnvironment) {
 	var badContainers []string
```

Comment:
The four tests use the same code in the loop, except for the capability string. Isn't it a good candidate for refactoring, putting the whole loop in a separate function, so it can be called like this?
```
func TestSysAdminCapability(env *provider.TestEnvironment) {
    badContainers := CheckForbiddenCapability(env.Containers, "SYS_ADMIN")
    testhelper.AddTestResultLog("Non-compliant", badContainers, tnf.ClaimFilePrintf, ginkgo.Fail)
}
```

Reply:
Yup I can do that.  👍

Reply:
Done.